### PR TITLE
Add Solr 5.5.2 and fix aliases

### PR DESCRIPTION
**Pull request by makuk66**
Announcement: http://mail-archives.apache.org/mod_mbox/www-announce/201606.mbox/%3c93C75552-DF74-4E0D-B02C-9B33B5E36587@apache.org%3e

Release Notes: https://lucene.apache.org/solr/5_5_2/changes/Changes.html

### CODE REVIEW

**Comment by tianon** [comment]
```diff
warning: insecure protocol git:// detected: git://github.com/docker-solr/docker-solr
diff --git a/solr_5.5.1-alpine/Dockerfile b/solr_5.5.2-alpine/Dockerfile
similarity index 92%
rename from solr_5.5.1-alpine/Dockerfile
rename to solr_5.5.2-alpine/Dockerfile
index 2aac9b7..033faa3 100644
--- a/solr_5.5.1-alpine/Dockerfile
+++ b/solr_5.5.2-alpine/Dockerfile
@@ -24,13 +24,13 @@ ENV SOLR_UID 8983
 RUN addgroup -S -g $SOLR_UID $SOLR_USER && \
   adduser -S -u $SOLR_UID -g $SOLR_USER $SOLR_USER
 
-ENV SOLR_KEY EDF961FF03E647F9CA8A9C2C758051CCA3A13A7F
+ENV SOLR_KEY 5F55943E13D49059D3F342777186B06E1ED139E7
 RUN gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$SOLR_KEY"
 ENV GPG_KEYSERVER ${GPG_KEYSERVER:-hkp://ha.pool.sks-keyservers.net}
 RUN gpg --keyserver "$GPG_KEYSERVER" --recv-keys "$SOLR_KEY"
 
-ENV SOLR_VERSION 5.5.1
-ENV SOLR_SHA256 3400b8aebde532e085a7cd888600e056f3fdc12ef889681516ccce9421926083
+ENV SOLR_VERSION 5.5.2
+ENV SOLR_SHA256 e62bab565675e10d27f40d5bb090b4181b2f0c21870adf98d1ea873ead7758e1
 ENV SOLR_URL ${SOLR_DOWNLOAD_SERVER:-http://www-us.apache.org/dist/lucene/solr}/$SOLR_VERSION/solr-$SOLR_VERSION.tgz
 
 RUN mkdir -p /opt/solr && \
diff --git a/solr_5.5.1/scripts/docker-entrypoint.sh b/solr_5.5.2-alpine/scripts/docker-entrypoint.sh
similarity index 100%
rename from solr_5.5.1/scripts/docker-entrypoint.sh
rename to solr_5.5.2-alpine/scripts/docker-entrypoint.sh
diff --git a/solr_5.5.1/scripts/wait-for-solr.sh b/solr_5.5.2-alpine/scripts/wait-for-solr.sh
similarity index 100%
rename from solr_5.5.1/scripts/wait-for-solr.sh
rename to solr_5.5.2-alpine/scripts/wait-for-solr.sh
diff --git a/solr_5.5.1/Dockerfile b/solr_5.5.2/Dockerfile
similarity index 92%
rename from solr_5.5.1/Dockerfile
rename to solr_5.5.2/Dockerfile
index 65a6362..6cab8a6 100644
--- a/solr_5.5.1/Dockerfile
+++ b/solr_5.5.2/Dockerfile
@@ -20,12 +20,12 @@ ENV SOLR_UID 8983
 RUN groupadd -r -g $SOLR_UID $SOLR_USER && \
   useradd -r -u $SOLR_UID -g $SOLR_USER $SOLR_USER
 
-ENV SOLR_KEY EDF961FF03E647F9CA8A9C2C758051CCA3A13A7F
+ENV SOLR_KEY 5F55943E13D49059D3F342777186B06E1ED139E7
 ENV GPG_KEYSERVER ${GPG_KEYSERVER:-hkp://ha.pool.sks-keyservers.net}
 RUN gpg --keyserver "$GPG_KEYSERVER" --recv-keys "$SOLR_KEY"
 
-ENV SOLR_VERSION 5.5.1
-ENV SOLR_SHA256 3400b8aebde532e085a7cd888600e056f3fdc12ef889681516ccce9421926083
+ENV SOLR_VERSION 5.5.2
+ENV SOLR_SHA256 e62bab565675e10d27f40d5bb090b4181b2f0c21870adf98d1ea873ead7758e1
 ENV SOLR_URL ${SOLR_DOWNLOAD_SERVER:-http://www-us.apache.org/dist/lucene/solr}/$SOLR_VERSION/solr-$SOLR_VERSION.tgz
 
 RUN mkdir -p /opt/solr && \
diff --git a/solr_5.5.1-alpine/scripts/docker-entrypoint.sh b/solr_5.5.2/scripts/docker-entrypoint.sh
similarity index 100%
rename from solr_5.5.1-alpine/scripts/docker-entrypoint.sh
rename to solr_5.5.2/scripts/docker-entrypoint.sh
diff --git a/solr_5.5.1-alpine/scripts/wait-for-solr.sh b/solr_5.5.2/scripts/wait-for-solr.sh
similarity index 100%
rename from solr_5.5.1-alpine/scripts/wait-for-solr.sh
rename to solr_5.5.2/scripts/wait-for-solr.sh
```

File: library/solr [hunk 1]
```diff
@@ -13,24 +13,26 @@
...
 
 6.1.0: git://github.com/docker-solr/docker-solr@1c2f9a0791e854c05c41952d5a086d57ebadc5f2 6.1
 6.1: git://github.com/docker-solr/docker-solr@1c2f9a0791e854c05c41952d5a086d57ebadc5f2 6.1
+6: git://github.com/docker-solr/docker-solr@1c2f9a0791e854c05c41952d5a086d57ebadc5f2 6.1
+latest,: git://github.com/docker-solr/docker-solr@1c2f9a0791e854c05c41952d5a086d57ebadc5f2 6.1
 
 6.1.0-alpine: git://github.com/docker-solr/docker-solr@1c2f9a0791e854c05c41952d5a086d57ebadc5f2 6.1/alpine
 6.1-alpine: git://github.com/docker-solr/docker-solr@1c2f9a0791e854c05c41952d5a086d57ebadc5f2 6.1/alpine
+6-alpine: git://github.com/docker-solr/docker-solr@1c2f9a0791e854c05c41952d5a086d57ebadc5f2 6.1/alpine
+latest,-alpine: git://github.com/docker-solr/docker-solr@1c2f9a0791e854c05c41952d5a086d57ebadc5f2 6.1/alpine
```
Comment (tianon) [inline library/solr]: I think you've got some formatting issues here (and above on line `33` where we've got `latest,:` :smile:).

Comment (tianon) [inline library/solr]: (FWIW, running your `./generate-stackbrew-library.sh` script from a local checkout appears to have the expected output :innocent:)

Comment (makuk66) [inline library/solr]: Problem was https://github.com/docker-solr/docker-solr/commit/9ccca5074ea1ec659e223be48d8cecb6dcc7616c

**Comment by yosifkit** [comment]
LGTM, Build test of #1887; 47aeb8f (`solr`):

```console

$ bashbrew build solr:5.3.2
Using bashbrew/cache:5007541fd23c4534b4807913cc4710124a30f57c9f0c01756f5618e2c7852673 (solr:5.3.2)
Tagging solr:5.3.2
Tagging solr:5.3

$ test/run.sh solr:5.3.2
testing solr:5.3.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build solr:5.3.2-alpine
Using bashbrew/cache:5971bfac70eaf0705732c2c78c766e9269a52811c5e1730677e4d18e5d5ed12d (solr:5.3.2-alpine)
Tagging solr:5.3.2-alpine
Tagging solr:5.3-alpine

$ test/run.sh solr:5.3.2-alpine
testing solr:5.3.2-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build solr:5.4.1
Using bashbrew/cache:f383fff718720caef088f91b1a4e03ac46584408bb5f4a1ab24eb41aabd8089b (solr:5.4.1)
Tagging solr:5.4.1
Tagging solr:5.4

$ test/run.sh solr:5.4.1
testing solr:5.4.1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build solr:5.4.1-alpine
Using bashbrew/cache:3905024d88dbdae3d5c049b6b6500a41d3a815e62f477813c89070414f1cc823 (solr:5.4.1-alpine)
Tagging solr:5.4.1-alpine
Tagging solr:5.4-alpine

$ test/run.sh solr:5.4.1-alpine
testing solr:5.4.1-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build solr:5.5.2
warning: insecure protocol git:// detected: git://github.com/docker-solr/docker-solr
Building bashbrew/cache:9a0ca2c0a2958c3e9ed95a6fd1cc77173ecb3ce2916b8746cb418efcde8b3808 (solr:5.5.2)
Tagging solr:5.5.2
Tagging solr:5.5
Tagging solr:5

$ test/run.sh solr:5.5.2
testing solr:5.5.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build solr:5.5.2-alpine
Using bashbrew/cache:1831240c69147d12eeef4bf552df951a1e55f875130d4c05c43bd80930548372 (solr:5.5.2-alpine)
Tagging solr:5.5.2-alpine
Tagging solr:5.5-alpine
Tagging solr:5-alpine

$ test/run.sh solr:5.5.2-alpine
testing solr:5.5.2-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build solr:6.0.1
Using bashbrew/cache:cbe048b4e449085528e37e82138654d55d5b507b9dcd1824317c31eabbf44f1b (solr:6.0.1)
Tagging solr:6.0.1
Tagging solr:6.0

$ test/run.sh solr:6.0.1
testing solr:6.0.1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build solr:6.0.1-alpine
Using bashbrew/cache:187c890b6468026fb2cf3544e7eb27cce707f713578bcd25d6de32d67e852337 (solr:6.0.1-alpine)
Tagging solr:6.0.1-alpine
Tagging solr:6.0-alpine

$ test/run.sh solr:6.0.1-alpine
testing solr:6.0.1-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build solr:6.1.0
Using bashbrew/cache:b9d803fbc01b8d5f45752ac47f51b143fb8abaa3aa326e90f84c72cf1cc2a873 (solr:6.1.0)
Tagging solr:6.1.0
Tagging solr:6.1
Tagging solr:6
Tagging solr:latest

$ test/run.sh solr:6.1.0
testing solr:6.1.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build solr:6.1.0-alpine
Using bashbrew/cache:4b082dfbb884465f4563324486d0efff4069190bb541575dc74a37a345f81d78 (solr:6.1.0-alpine)
Tagging solr:6.1.0-alpine
Tagging solr:6.1-alpine
Tagging solr:6-alpine
Tagging solr:alpine

$ test/run.sh solr:6.1.0-alpine
testing solr:6.1.0-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

File: library/solr (hunk 1 above)
Comment (tianon) [inline library/solr]: Ahaha, win! :metal:

**Pull request merged**